backends/platform/iphone/iphone_common.h

```diff
@@ -52,6 +53,11 @@ enum UIViewSwipeDirection {
 	kUIViewSwipeRight = 8
 };
 
+enum UIViewTapDescription {
+	kUIViewTapSingle = 1,
+	kUIViewTapDouble = 2,
```

Comment:
This has an extra "," at the end which is only legal in C++11, which we do not use.